presto-ui/src/webpack.config.js

```diff
@@ -127,4 +120,37 @@ module.exports = (env) => {
             },
         },
     };
+
+    const spaConfig = {
+        ...baseConfig,
+        plugins: [
+            ...baseConfig.plugins,
+            new HtmlWebpackPlugin({
+                inject: 'body',
+                filename: path.join(__dirname, '..', outputDir, 'dev', 'query_viewer_spa.html'),
```

Comment:
Should this be simplified to below?
```suggestion
                filename: path.join( 'dev', 'query_viewer_spa.html'),
```

Reply:
@yhwang Thanks, updated